slang-checks/src/main/java/org/sonarsource/slang/checks/CheckList.java

```diff
@@ -43,7 +43,9 @@
     VariableAndParameterNameCheck.class
   };
 
-  private static final Class[] KOTLIN_CHECK_BLACK_LIST = {};
+  private static final Class[] KOTLIN_CHECK_BLACK_LIST = {
+    OctalValuesCheck.class
```

Comment:
Please create a ticket to enable this rule in Kotlin, so we don't forget about it

Reply:
Kotlin [does not support](https://kotlinlang.org/docs/reference/basic-types.html) octal literals:
> NOTE: Octal literals are not supported.

Reply:
OK, I was not aware, then maybe we can put a comment here?